tests/data/conf_covariance_clusters.yaml

```diff
@@ -0,0 +1,31 @@
+tjpcov:
+  use_mpi: True
```

Comment:
The test is failing because you have `use_mpi: True` in the "serial" tests. At that stage, mpi has not been installed

Reply:
Derp. Sorry I took so long to look at this!  Just pushed the trivial changes.